scripts/installer.sh

```diff
@@ -501,10 +493,13 @@ main() {
 				legacy)
 					$CURL "https://pkgs.tailscale.com/$TRACK/$OS/$VERSION.asc" | $SUDO apt-key add -
 					$CURL "https://pkgs.tailscale.com/$TRACK/$OS/$VERSION.list" | $SUDO tee /etc/apt/sources.list.d/tailscale.list
+					$SUDO chmod 0644 /etc/apt/sources.list.d/tailscale.list
```

Comment:
I think we should download these files to a mktemp and then use install(1) to install them. This would cleanup two things, one that we set the mode correctly just at install time, and secondly that a broken or partial transfer does not result in a broken package manager.

Reply:
Thanks, I'll take a look at doing that in a separate PR